build fails when blackmagic is submodule: command.c:34:10: fatal error: version.h: No such file or directory

For some reason, I'm unable to build blackmagic on mac:

```
blackmagic on  imxrt-debug-format-fix 
❯ make V=1                                                                                                                                         (base) 
if [ ! -f libopencm3/Makefile ]; then \
		echo "Initialising git submodules..." ;\
		git submodule init ;\
		git submodule update ;\
	fi
/Library/Developer/CommandLineTools/usr/bin/make  -C libopencm3 lib/stm32/f1 lib/stm32/f4 lib/lm4f
rm -f .stamp_failure_lib_stm32_f1
  BUILD   lib/stm32/f1
/Library/Developer/CommandLineTools/usr/bin/make --directory=lib/stm32/f1 PREFIX="arm-none-eabi-" || \
		echo "Failure building: lib/stm32/f1: code: $?" > .stamp_failure_lib_stm32_f1
make[2]: Nothing to be done for `all'.
rm -f .stamp_failure_lib_stm32_f4
  BUILD   lib/stm32/f4
/Library/Developer/CommandLineTools/usr/bin/make --directory=lib/stm32/f4 PREFIX="arm-none-eabi-" || \
		echo "Failure building: lib/stm32/f4: code: $?" > .stamp_failure_lib_stm32_f4
make[2]: Nothing to be done for `all'.
rm -f .stamp_failure_lib_lm4f
  BUILD   lib/lm4f
/Library/Developer/CommandLineTools/usr/bin/make --directory=lib/lm4f PREFIX="arm-none-eabi-" || \
		echo "Failure building: lib/lm4f: code: $?" > .stamp_failure_lib_lm4f
make[2]: Nothing to be done for `all'.
/Library/Developer/CommandLineTools/usr/bin/make  -C src
Git not found, assuming up to date include/version.h
  CC      command.c
arm-none-eabi-gcc -Wall -Wextra -Werror -Wreturn-type -Wno-char-subscripts -std=c11 -g3 -MD -I./target -I. -Iinclude -Iplatforms/native -Istm32/include -mcpu=cortex-m3 -mthumb -DSTM32F1 -DBLACKMAGIC -I../libopencm3/include -Iplatforms/stm32 -DDFU_SERIAL_LENGTH=9 -Os -DPC_HOSTED=0 -Iplatforms/common -c command.c -o command.o
command.c:34:10: fatal error: version.h: No such file or directory
   34 | #include "version.h"
      |          ^~~~~~~~~~~
compilation terminated.
make[1]: *** [command.o] Error 1
make: *** [all] Error 2

blackmagic on  imxrt-debug-format-fix 
❯                                                                                                                                                  (base) 
```

--- a/src/Makefile
+++ b/src/Makefile
@@ -145,7 +145,7 @@ endif
 .PHONY:	clean host_clean all_platforms clang-format FORCE
 
 
-GIT_VERSION := $(shell if [ -d "../.git" ]; then git describe --always --dirty --tags; fi)
+GIT_VERSION := $(shell if [ -e "../.git" ]; then git describe --always --dirty --tags; fi)
 
 clean:	host_clean
 	$(Q)echo "  CLEAN"

